static/app/utils/profiling/flamegraphChart.tsx

```diff
@@ -115,14 +118,14 @@ export class FlamegraphChart {
         }
 
         // Track and update X domain max and min
-        if (m.elapsed_since_start_ns > this.domains.x[1]) {
-          this.domains.x[1] = m.elapsed_since_start_ns;
+        if (m.elapsed > this.domains.x[1]) {
+          this.domains.x[1] = m.elapsed;
```

Comment:
this had previously shared the type with the measurement, but was now moved to a separate field